app/Http/Controllers/TotalPrices/ProcessController.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace App\Http\Controllers\TotalPrices;
+
+use App\Actions\ElectricityPrices\RetrieveSpotPrices;
+use App\Actions\ElectricityPrices\RetrieveTariffFromOperator;
+use App\Http\Controllers\Controller;
+use App\Models\GridOperatorNettariffProperty;
+use App\Models\Operator;
+use Carbon\Carbon;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+
+class ProcessController extends Controller
```

Comment:
@Tvup 
Hvis jeg skal være ærlig, så ville jeg nok normalt sige at den her controller er alt for stor.

Alle de private metoder der er, mener jeg burde flyttes til en service class i stedet for - og så kan den service class bare dependency injectes ind i controlleren. 

Det samme med `RetrieveSpotPrices` - den kan også bare denpency injectes istedet for at lave en ny instans af den flere gange.

Hvis det blev flyttet til en service class - vil det også være en del nemmere at skrive tests :) 

Hvis du er frisk på det - Så kan vi merge det her PR, og så kan jeg starte på det, så kan du se hvad du syntes om det ? :)

Reply:
@negoziator 
Det skal du :)

Ja, det lyder også meget, som jeg egentlig ville have det. Jeg er dog ind i mellem begyndt på at indføre en klasse mellem service og controller, som jeg kalder "Action", fordi det ind i mellem kan synes som om, der er kode, der skal eksekveres men ikke naturligt hører hjemme i en controller.
Her er jeg endda begyndt at have opdelt en funktionalitet i to controllers, en til view og en til de ting, der f.eks. submittes af viewet. Jeg kan dog ikke finde ud af, hvordan man så skal navngive.

Oprindeligt boede en del af det her i Elcontroller, som er endt som den alt for store skraldespand - derfor synes jeg dette var en forbedring, selvom det kan gøres bedre endnu.

Ja, det er jeg helt frisk på :D

Reply:
Fedt manner! -- jeg har faktisk set andre bruge den taktik andre steder - altså med at bruge Action -classes, og jeg har altid syntes det virker smart - jeg har bare selv svært ved at få det gjort eller afprøvet, så det ville være oplagt at prøve det af og se hvor det kan bære os hen 😄 

Okay, jamen er det så ikke bare noget med at du merger det her PR - og så tager vi den derfra? 💪🏼 

Mine kommentarer til resten vil være meget det samme alligevel 😄

Reply:
@negoziator Så det er det, jeg gør nu :)